templates/support/index.html

```diff
@@ -516,13 +516,14 @@ <h5>10 years of coverage for 36,000+ packages</h5>
       <div class="col">
         <h5>Get phone and ticket support with Ubuntu Pro + support</h5>
         <p>
-          Build with confidence with 24/7/365 phone and ticket support. Get prompt help when something breaks on more than 36,000 packages in the Ubuntu Main and Universe repositories, including the most widely used open source applications and toolchains. Our 24/7 plans now include SLAs not only for initial response times, but also for ongoing follow-up updates ensuring continuous visibility and faster remediation throughout the lifecycle of your support case.
+          Build with confidence with 24/7/365 phone and ticket support. Get prompt help when something breaks on any of the packages in the Ubuntu Main and Universe repositories, including the most widely used open source applications and toolchains. Our 24/7 plans now include SLAs not only for initial response times, but also for ongoing follow-up updates ensuring continuous visibility and faster remediation throughout the lifecycle of your support case.
         </p>
         <p>
           Benefit from the collective knowledge of our team, based on our 20 years of experience building and sustaining open source software.
         </p>
         <a class="p-button--positive"
            href="/support/contact-us?product=support-overview">Get in touch</a>
+        <a href="https://assets.ubuntu.com/v1/5c454478-ubuntu_pro_and_support_data_sheet.pdf">See how Ubuntu Pro + Support stands out&nbsp;&rsaquo;</a>
```

Comment:
Small typo fix from [copydoc](https://docs.google.com/document/d/1AU2dwTXpQk2UuEvSuAqRKrbPWYb-LR0ZGvWu6xw95I8/edit?disco=AAABtDY7X4s)
```suggestion
        <a href="https://assets.ubuntu.com/v1/5c454478-ubuntu_pro_and_support_data_sheet.pdf">Discover how Ubuntu Pro + Support stands out&nbsp;&rsaquo;</a>
```